drizzlepac/align.py

```diff
@@ -694,6 +707,7 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, align_par
     overall_comp = False
 
     do_consistency_check = align_pars['determine_fit_quality'].get('consistency_check', True)
```

Comment:
Is "consistency_check" an obsolete or deprecated value in the parameter files as it does not currently appear in any of the json parameter files.  I understand there is a default value here, but are there times when we do not want the consistency check done?

Ok, I looked a bit more after I got confused between the use of "do_consistency_check" and "consistency_check".  Is "consistency_check" really "is_consistent"?

Reply:
The parameter 'consistency_check' here was renamed to 'do_consistency_check' in db1f6d1 to avoid any future confusion with the variable of the same name and different meaning later in the function.